### PR TITLE
[10.0.1] Upgrade to SDK version 10.0.1

### PR DESCRIPTION
**Pull request by serenastorm**

### CODE REVIEW

File: nextjs/src/components/StorytellerClipsGridView.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      columns: 4,
+    },
+  },
+};
+
+interface StorytellerClipsGridViewProps {
+  collection: string;
+  title?: string | undefined;
+  moreButtonTitle?: string | undefined;
+  displayLimit?: number | undefined;
```
Comment (pawel-skiba) [inline nextjs/src/components/StorytellerClipsGridView.tsx:23]: It should be enough:

```
title?: string;
moreButtonTitle?: string;
displayLimit?: number;
```

File: nextjs/src/components/StorytellerClipsRowView.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import buildBasicTheme from '@/helpers/buildBasicTheme';
+import useStoryteller from '@/hooks/useStoryteller';
+import TitleAndMoreButton from '@/components/TitleAndMoreButton';
+
+interface StorytellerStoriesRowViewProps {
+  size: keyof typeof Size;
+  collection: string;
+  title?: string | undefined;
+  moreButtonTitle?: string | undefined;
+  displayLimit?: number | undefined;
```
Comment (pawel-skiba) [inline nextjs/src/components/StorytellerClipsRowView.tsx:18]: Same as before

File: nextjs/src/components/StorytellerClipsGridView.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import {
+  StorytellerClipsGridView as GridView,
+  UiTheme,
+} from '@getstoryteller/storyteller-sdk-javascript';
+import { useEffect, useRef } from 'react';
+import useStoryteller from '@/hooks/useStoryteller';
+import TitleAndMoreButton from '@/components/TitleAndMoreButton';
```
Comment (pawel-skiba) [inline nextjs/src/components/StorytellerClipsGridView.tsx:9]: We could reorder to keep react imports on top as in other files

File: nextjs/src/components/StorytellerListView.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  // These categories are configured in the Storyteller CMS.
+  const categories =
+    list.categories?.map((category) =>
+      category.replace('[LANGUAGE]', languageSetting),
+    ) || [];
+
+  const isRow = list.layout === Layout.row;
+  const isGrid = list.layout === Layout.grid;
+  const isStoriesList = list.videoType === VideoType.stories;
+  const isClipsList = list.videoType === VideoType.clips;
```
Comment (pawel-skiba) [inline nextjs/src/components/StorytellerListView.tsx:50]: Same comment as in another PR. I am not sure if videoType fully indicates what this content is, but it's a matter of discussion, so not a blocker.

File: nextjs/src/components/TitleAndMoreButton.tsx [hunk 5]
```diff
@@ -1,24 +1,28 @@
 'use client';
 
 import Link from 'next/link';
+import type { Url } from 'next/dist/shared/lib/router/router';
 
 interface Props {
   title?: string | undefined;
   moreButtonTitle?: string | undefined;
```
Comment (pawel-skiba) [inline nextjs/src/components/TitleAndMoreButton.tsx:8]: Same as before. We don't need undefined when ? used.

File: nextjs/src/components/StorytellerListView.tsx (hunk 4 above)
Comment (serenastorm) [reply nextjs/src/components/StorytellerListView.tsx:50]: This is also coming from the API

File: nextjs/src/components/StorytellerClipsGridView.tsx (hunk 1 above)
Comment (serenastorm) [reply nextjs/src/components/StorytellerClipsGridView.tsx]: Ah yes, I copied from another file and didn't even notice. I'll update everywhere.

**Pull request merged**